tasks/gulp-build.js

```diff
@@ -18,5 +34,95 @@ gulp.task('build:dev:watch', ['build:dev'], () => {
 });
 
 gulp.task('build:prod', ['babel:common', 'prepare:staticNewStuff'], (done) => {
-  gulp.start('grunt-build:prod', done);
+  var files = JSON.parse(fs.readFileSync('./website/public/manifest.json'));
+  var uglifyFiles = {};
+  var cssminFiles = {};
+
+  _.each(files, function(val, key) {
+    var js = uglifyFiles['website/build/' + key + '.js'] = [];
+
+    _.each(files[key].js, function(val){
+      var path = "./";
+      if( val.indexOf('common/') == -1)
+        path = './website/public/';
+      js.push(path + val);
+    });
+
+    var css = cssminFiles['website/build/' + key + '.css'] = [];
+
+    _.each(files[key].css, function(val){
+      var path = "./";
+      if( val.indexOf('common/') == -1) {
+        path = (val == 'app.css' || val == 'static.css') ?  './website/build/' : './website/public/';
+      }
+      css.push(path + val)
+    });
+  
+  });
+
+  doBrowserify();
+  doUglify(uglifyFiles);
+  doCssmin(cssminFiles);
+  doStylus();
+  gulp.start('copy');
+  gulp.start('hashres');
 });
+
+
+function doBrowserify() {
```

Comment:
At first I thought if would be easier, but thinking about it. It's probable easier to read if I make it a gulp task. I'll refactor it. And the style here is one file per gulp task?